src/style.css

```
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+
+}
```

```suggestion
  display: flex;
}
```
Please fix all cases